src/components/files/hooks/useGetFileDownloadUrl.ts

```diff
@@ -0,0 +1,45 @@
+import { useApolloClient } from '@apollo/client';
+import { useSnackbar } from 'notistack';
+import { useCallback } from 'react';
+import {
+  FileDownloadUrl_Directory_Fragment,
+  FileNodeDownloadUrlDocument,
+  FileNodeDownloadUrlQuery,
+} from '../FileActions';
+
+export const useGetFileDownloadUrl = () => {
+  const { enqueueSnackbar } = useSnackbar();
+
+  const showSnackbarError = useCallback(() => {
+    enqueueSnackbar('Could not download file', {
+      variant: 'error',
+    });
+  }, [enqueueSnackbar]);
+
+  const client = useApolloClient();
+
+  const getFileDownloadUrl = useCallback(
+    async (id: string): Promise<string> => {
+      try {
+        const { data } = await client.query<FileNodeDownloadUrlQuery>({
+          query: FileNodeDownloadUrlDocument,
+          variables: { id },
+          fetchPolicy: 'network-only',
```

Comment:
Can you add a comment saying this is to prevent downloads requests from being stale? I don't want to wonder why we are doing this 6 months from now :)